### PR TITLE
Fixed content change log regression

### PR DESCRIPTION
**Pull request by abubakarA-Dot**

### CODE REVIEW

**Comment by enoren5** [comment]
The previous state of the for loop for this changelog template looked like this:
```
   {% for change in changes %}
 
       <center><h1>{{ change.title }}</h1></center>
       <ul>
       {% for point in change.log_to_bullets %}
           <li>{{point}}</li>
       {% endfor %}
       </ul>
 
   {% endfor %}
```
The HTML and content generated looked like this:
```
     <center><h1>Content Changes</h1></center>
        <ul>
        
            <li>Cras hendrerit sit amet nulla sit amet bland</li>
        
            <li>it. Interdum et malesuada fames ac ante ipsum primis in faucibus. Integer leo quam, lacinia vitae to</li>
        
            <li>rtor in, semper laoreet purus. In vestibulum, nunc quis rutrum scelerisque, massa felis hendrerit</li>
        
            <li> metus, vitae pulvinar erat metus at massa. Sed auctor vel eros sed rhoncus. Duis mattis nunc augue, sit amet commodo metus laoreet vel. Vestibu</li>
        
            <li>lum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia curae; Morbi eu ex id dolor pharetr</li>
        
            <li>a lobortis. Curabitur sit amet purus quis felis ultrices ultricies et eu lectus. Aenean consectetur sollicitudin diam vel molestie.</li>
        
        </ul>
```

Notice the unordered list and each list item?

It parsed like this:

For Abu -content changes bullets showing properly in earlier state-Screenshot_2022-03-22_15-47-42

The above is how it should look.

However your Jinja for loop parses the list in paragraph tags, causing only a single bullet point to appear at the top:

For Abu - content change log no bullets -Screenshot_2022-03-22_15-56-33

This needs to be corrected so that Jinja actually automatically builds an unordered list (like it did before).

This problem (regression) involving content change log output is also present at this location:  `https://127.0.0.1:8000/article/all_content_dump/`

To summarize, you need to use Jinja to build the unordered list properly for both places where Django serves the content changelog:
- `https://127.0.0.1:8000/article/all_content_dump/`
- `https://127.0.0.1:8000/article/content_changelog/`

I found [the official Django doc for builtins which covers the `linebreaks` keyword](https://docs.djangoproject.com/en/4.0/ref/templates/builtins/#linebreaks) you used but there is no similar builtin for generating unordered lists. So we will have to find some other way of building unordered lists as summarized above without using a builtin.
